tests/rest/client/v1/utils.py

```diff
@@ -152,8 +161,12 @@ def send_state(self, room_id, event_type, body, tok, expect_code=200, state_key=
             path = path + "?access_token=%s" % tok
 
         request, channel = make_request(
-            self.hs.get_reactor(), "PUT", path, json.dumps(body).encode("utf8")
+            self.hs.get_reactor(),
+            method,
+            path,
+            content=json.dumps(body).encode("utf8") if body else b"",
```

Comment:
This fails if `body` is `None`. I suggest:

```python
content = b""
if body is not None:
    content = json.dumps(body).encode("utf8")

request, channel = make_request(
    self.hs.get_reactor(), method, path, content
)
```

Reply:
I think it's failing because `body` being `{}` is falsey?

Reply:
Your code works though :)